components/unit/UnitQuery.vue

```diff
@@ -115,7 +115,7 @@
           </template>
           <template v-else>
             <VRow>
-              <VCol>
+              <VCol style="text-align: center">
```

Comment:
It's better if we can avoid using inline styles, they make it harder to have a consistent look everywhere. In this case, you should be able to use [justify from the vuetify grid system](https://vuetifyjs.com/en/components/grids/#justify)

Reply:
If I use something like this:

```
<VRow justify="center">
  <VCol cols="4">
    <p><i>No relevant data found</i></p>
  </VCol>
</VRow>
```

It centers the column but not the text inside it:

![Screenshot_20211124_094415](https://user-images.githubusercontent.com/25420002/143204318-2b391aed-26e9-439d-adc2-a6ccba290b97.png)

I would say that it doesn't look as good as having the text fully centered, what do you think ? Can we achieve this without using `style="text-align: center"` ?

Reply:
It looks fine to me. I don't like centered text. 

And the text in the description is also left-justified (could be something I did, actually).

Reply:
text-center is also an acceptable solution